spec/importers/file_importers/students_importer_spec.rb

```diff
@@ -5,7 +5,7 @@
   describe '#import_row' do
 
     context 'good data' do
-      let(:file) { File.open("#{Rails.root}/spec/fixtures/fake_students_export.txt") }
+      let(:file) { File.read("#{Rails.root}/spec/fixtures/fake_students_export.txt") }
```

Comment:
This was a test bug before, the `CsvDownloader` code passes a string of the CSV file to the transformer class.  But because of the way `transform` was implemented before, it just so happened that passing a file in the test would work as well.  This isn't the case anymore with these changes, so this fixes the test bug to correctly pass a string.